web/src/settings_org.js

```diff
@@ -784,6 +784,7 @@ function set_up_dropdown_widget(setting_name, setting_options, setting_type) {
         on_mount_callback(dropdown) {
             if (setting_type === "group") {
                 $(dropdown.popper).css("min-width", "300px");
```

Comment:
Should we also set the min width to match the min width of button which opens the dropdown.

Reply:
@amanagr thoughts on this?

Reply:
No, since we want dropdown to we completely visible at 320px which is the minimum width we actively support.